components/api_calls/rpms_images_fetcher_calls.js

```diff
@@ -0,0 +1,19 @@
+import axios from 'axios';
+
+let server_endpoint = null
+
+if (process.env.NEXT_PUBLIC_RUN_ENV === "dev") {
+    server_endpoint = "http://localhost:8080/"
```

Comment:
I think the variable naming convention for variables is camelCase. Think we didn't follow them till now, but it might be a good idea for new code to follow conventions so that it'll be easier for us to maintain code quality. Also there's a convention for file names as well. Actually can you setup a linter like `ESlint`, so that its automatically enforsed?